### PR TITLE
Adds MoreIcon

### PR DESCRIPTION
**Pull request by sweir27**
This adds the `MoreIcon` which we'll use to hide admin actions on the artwork page.

Also added it to the `Icons` doc!

image

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit 82872cf0cb96ef66c74a182495f51255ceb93b1f

https://deploy-preview-237--artsy-palette.netlify.com

File: src/svgs/MoreIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from "react"
+
+interface MoreIconProps {
+  fill?: string
```
Comment (damassi) [inline src/svgs/MoreIcon.tsx]: Is this being used?

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit efa1b0eb78207ef53b99228395612b06c4ded045

https://deploy-preview-237--artsy-palette.netlify.com

**Comment by damassi** [comment]
👍

**Comment by artsyit** [comment]
:rocket: PR was released in v2.36.1 :rocket:

**Pull request merged**